external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftQuickstartUsingS2iIT.java

```diff
@@ -9,7 +9,7 @@
 import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusSnapshot;
 import io.quarkus.test.services.GitRepositoryQuarkusApplication;
 
-@DisabledOnNative
+@DisabledOnNative(reason = "S2i is not supported on Native yet: https://github.com/quarkus-qe/quarkus-test-framework/issues/206")
```

Comment:
I think S2I for native needed some special Quarkus image which was supported just on community side. On product side native build using S2I wasn't supported. 

We used that community image for Native S2I in OpenShiftTS. @mjurc can you point to the details?
Ideally update https://github.com/quarkus-qe/quarkus-test-framework/issues/206